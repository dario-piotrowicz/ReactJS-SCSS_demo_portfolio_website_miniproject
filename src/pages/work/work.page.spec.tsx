import React from 'react';
import { shallow } from 'enzyme';
import WorkPage from './work.page.component';

describe('work page', () => {
  it('should match its snapshot', () => {
    expect(shallow(<WorkPage />)).toMatchSnapshot();
  });
});
