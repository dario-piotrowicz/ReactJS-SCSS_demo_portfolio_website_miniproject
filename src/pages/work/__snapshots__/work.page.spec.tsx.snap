// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`work page should match its snapshot 1`] = `
<Fragment>
  <GallerySection />
</Fragment>
`;
